Lib/test/test_build_details.py

```diff
@@ -147,5 +174,64 @@ def test_c_api(self):
         self.assertTrue(os.path.exists(os.path.join(value['pkgconfig_path'], f'python-{version}.pc')))
 
 
+@unittest.skipIf(
+    generate_build_details is None,
+    "Failed to import generate-build-details"
+)
+@unittest.skipIf(os.name != 'posix', 'Feature only implemented on POSIX right now')
+@unittest.skipIf(is_wasi or is_emscripten, 'Feature not available on WebAssembly builds')
```

Comment:
Thanks! I merged the backport: https://github.com/python/cpython/pull/138643

```suggestion
@unittest.skipIf(is_wasm32, 'Feature not available on WebAssembly builds')
```